src/components/ReportingConfig/ReportingConfigForm.test.jsx

```diff
@@ -114,6 +114,23 @@ const createConfig = () => { };
 const updateConfig = () => { };
 
 describe('<ReportingConfigForm />', () => {
+  it('properly handles deletion of configs', () => {
+    const mock = jest.fn();
+    const wrapper = mount((
+      <ReportingConfigForm
+        config={config}
+        createConfig={createConfig}
+        updateConfig={updateConfig}
+        availableCatalogs={availableCatalogs}
+        reportingConfigTypes={reportingConfigTypes}
+        deleteConfig={mock}
+      />
+    ));
+    // It's finding three buttons for some reason??
```

Comment:
yeah I am coming to prefer react testing library for this reasno, where it lets us focus more on user-visible or user-interactible pieces rather than dom details like this. but yeah not part of scope for this PR!

Reply:
in that setup, this line would ideally be sth like `userEvent.click(screen.findByRole('submit-button'))` which is also more a11y friendly

Reply:
agreed, although it's not the submit button, but the delete one 😅